packages/jest-cli/src/lib/validatePattern.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+const validatePattern = (pattern: string) => {
+  let isValid = true;
+
+  if (pattern) {
+    try {
+      // eslint-disable-next-line no-new
+      new RegExp(pattern, 'i');
+    } catch (e) {
+      isValid = false;
```

Comment:
can you just `return false` here and `return true` on the bottom? That way you don't need a variable.